### PR TITLE
Change UI on tool and use reviewed strings

### PR DESCRIPTION
**Pull request by wli3**
- Change where manifest path is put and remove stack trace

- Remove unused targetframework

- Add Json parsing error

- Add aka.ms link for missing command

- Remove not used resources and use reviewed strings

- Change success message order and color

### CODE REVIEW

**Comment by wli3** [comment]
@KathleenDollard I realize command conflicts error message could be multiple instead of only 2. So i changed it to the following

```
Command name conflicts. Command names are case insensitive.
	A command "dotnetsay" in package "local.tool.console.a"
        A command "dotnetSay" in package "command.name.collision.with.package.a
```

**Comment by wli3** [comment]
@dotnet/dotnet-cli add some context. I seat together with @KathleenDollard for 2 hours and reviewed all the strings for local tools and the related user experience. The end result is a lot of strings being changed. This PR 100 files. But it is all about strings, reorder message and different colors. I hope to get it in soon, since big string-loc PR like this would be very hard to resolve if there is any merge conflict

File: src/dotnet/ToolManifest/LocalizableStrings.resx [hunk 1]
```diff
@@ -118,32 +118,33 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="CannotFindAnyManifestsFileSearched" xml:space="preserve">
-    <value>Cannot find any manifests file. Searched {0}</value>
+    <value>Cannot find any manifests file. Searched
```
Comment (livarcocc) [inline src/dotnet/ToolManifest/LocalizableStrings.resx]: Just want to confirm the line break here is intentional. If that's the case, should we have some : separator or something like that.

File: src/dotnet/ToolManifest/LocalizableStrings.resx [hunk 2]
```diff
@@ -118,32 +118,33 @@
...
+{0}</value>
   </data>
   <data name="FieldCommandsIsMissing" xml:space="preserve">
-    <value>Field commands is missing.</value>
+    <value>Missing 'commands' entry.</value>
   </data>
   <data name="InvalidManifestFilePrefix" xml:space="preserve">
-    <value>Invalid manifest file.
-{0}</value>
+    <value>Invalid manifest file. Path: {0}
```
Comment (livarcocc) [inline src/dotnet/ToolManifest/LocalizableStrings.resx]: Seems to have a line break here as well.

File: src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx [hunk 3]
```diff
@@ -139,7 +139,7 @@
     <value>Add an additional NuGet package source to use during installation.</value>
   </data>
   <data name="CommandDescription" xml:space="preserve">
-    <value>Install a tool for use on the command line.</value>
+    <value>Install global tool or add tool to local manifest and make it available.</value>
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx]: @KathleenDollard to review. The end could be improved, I believe.

File: src/dotnet/commands/dotnet-tool/list/LocalizableStrings.resx [hunk 4]
```diff
@@ -136,7 +136,8 @@
     <value>(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</value>
   </data>
   <data name="InvalidPackageWarning" xml:space="preserve">
-    <value>Warning: tool package '{0}' is invalid: {1}</value>
+    <value>Warning: tool package '{0}' is invalid, please contact tool author:
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-tool/list/LocalizableStrings.resx]: Another line break.

File: src/dotnet/commands/dotnet-tool/restore/ToolRestoreCommand.cs [hunk 5]
```diff
@@ -85,7 +83,8 @@ public override int Execute()
             FilePath? customManifestFileLocation = GetCustomManifestFileLocation();
 
             FilePath? configFile = null;
-            if (_configFilePath != null) configFile = new FilePath(_configFilePath);
+            if (_configFilePath != null)
+                configFile = new FilePath(_configFilePath);
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-tool/restore/ToolRestoreCommand.cs]: NIT: open/close brackets

File: src/dotnet/commands/dotnet-tool/uninstall/LocalizableStrings.resx [hunk 6]
```diff
@@ -127,7 +127,7 @@
     <value>Please specify one tool Package Id to uninstall.</value>
   </data>
   <data name="CommandDescription" xml:space="preserve">
-    <value>Uninstall a tool from the current development environment.</value>
+    <value>Uninstall a global tool.</value>
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-tool/uninstall/LocalizableStrings.resx]: This one does not affect local tools as well?

File: src/dotnet/commands/dotnet-tool/update/xlf/LocalizableStrings.de.xlf [hunk 7]
```diff
@@ -23,8 +23,8 @@
         <note />
       </trans-unit>
       <trans-unit id="CommandDescription">
-        <source>Update a tool to the latest stable version.</source>
-        <target state="translated">Hiermit wird ein Tool auf die neueste stabile Version aktualisiert.</target>
+        <source>Update a global tool.</source>
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-tool/update/xlf/LocalizableStrings.de.xlf]: Same here.

File: src/dotnet/commands/dotnet-tool/uninstall/LocalizableStrings.resx (hunk 6 above)
Comment (wli3) [reply src/dotnet/commands/dotnet-tool/uninstall/LocalizableStrings.resx]: agree, we need to add local tools here

File: src/dotnet/commands/dotnet-tool/update/xlf/LocalizableStrings.de.xlf (hunk 7 above)
Comment (wli3) [reply src/dotnet/commands/dotnet-tool/update/xlf/LocalizableStrings.de.xlf]: We don't have update in the costing. It is a good to have feature. I think we should keep it global tools only for now

File: src/dotnet/ToolManifest/LocalizableStrings.resx (hunk 1 above)
Comment (wli3) [reply src/dotnet/ToolManifest/LocalizableStrings.resx]: : it good

File: src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx (hunk 3 above)
Comment (wli3) [reply src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx]: Install global or local tool. Local tools are added to manifest and restored.

File: src/dotnet/ToolManifest/LocalizableStrings.resx (hunk 1 above)
Comment (wli3) [reply src/dotnet/ToolManifest/LocalizableStrings.resx]: yes, the line break is intentional and there will be several indented new lines for detail. I added `:` at the end

File: src/dotnet/ToolManifest/LocalizableStrings.resx (hunk 2 above)
Comment (wli3) [reply src/dotnet/ToolManifest/LocalizableStrings.resx]: yes, it is intentional

File: src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx (hunk 3 above)
Comment (wli3) [reply src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx]: I used the string above. I worked with Kathleen to come up with it

File: src/dotnet/commands/dotnet-tool/list/LocalizableStrings.resx (hunk 4 above)
Comment (wli3) [reply src/dotnet/commands/dotnet-tool/list/LocalizableStrings.resx]: Yes, intentional

File: src/dotnet/commands/dotnet-tool/restore/ToolRestoreCommand.cs (hunk 5 above)
Comment (wli3) [reply src/dotnet/commands/dotnet-tool/restore/ToolRestoreCommand.cs]: fixed

File: src/dotnet/commands/dotnet-tool/uninstall/LocalizableStrings.resx (hunk 6 above)
Comment (wli3) [reply src/dotnet/commands/dotnet-tool/uninstall/LocalizableStrings.resx]: Added local tools

File: src/dotnet/commands/dotnet-tool/update/xlf/LocalizableStrings.de.xlf (hunk 7 above)
Comment (wli3) [reply src/dotnet/commands/dotnet-tool/update/xlf/LocalizableStrings.de.xlf]: I didn't change here

**Comment by wli3** [comment]
@livarcocc I addressed the comment, it is good to merge?

**Comment by wli3** [comment]
@dotnet-bot test Debian8.2 x64 Debug Build

**Comment by wli3** [comment]
@dotnet-bot test opensuse.42.3 x64 Debug Build

**Comment by livarcocc** [comment]
@dotnet-bot Test opensuse.42.3 x64 Debug Build

**Comment by wli3** [comment]
@dotnet-bot test opensuse.42.3 x64 Debug Build

**Comment by wli3** [comment]
@dotnet-bot test opensuse.42.3 x64 Debug Build

**Comment by wli3** [comment]
@dotnet-bot test opensuse.42.3 x64 Debug Build

**Pull request merged**